pkg/resource/parameter_group/update_pre_build_request.go

```diff
@@ -15,6 +15,8 @@ package parameter_group
 
```

Comment:
Why was this file renamed to `update_pre_build_request.go`? maybe better to just name it `hooks.go`?

Reply:
Because I wanted to make sure those file names match their hook names(hook call functions from file). Do you think it is better to put all functions into a hooks.go file?

Reply:
If merging all functions into a single hooks.go file, the pro is that only one hook file will show up, and the con is that it might be hard to find a specific function which is called by a specific template(hook).

Reply:
I found aws-rds team uses only one hooks.go file to include all functions. Currently aws-memorydb controller uses separate files for functions that hooks use. Should I merge them into one hooks.go file?

Reply:
@kyriechen96 yeah, that's currently the practice that most of the controllers follow...a single `pkg/$resource/hooks.go` file.

Reply:
Got it!